lib/mindwendel_web/controllers/admin/brainstorming_controller.ex

```diff
@@ -34,7 +39,8 @@ defmodule MindwendelWeb.Admin.BrainstormingController do
   end
 
   def update(conn, %{"id" => id, "brainstorming" => brainstorming_params}) do
-    brainstorming = Brainstormings.get_brainstorming_by!(%{admin_url_id: id})
+    brainstorming =
+      Brainstormings.get_brainstorming_by!(%{admin_url_id: id}) |> Repo.preload(:labels)
```

Comment:
here you preload without the where clause (see above, edit) - which one is the better one? It should be consistent

Reply:
You are right. I will align it.